interface/main/left_nav.php

```diff
@@ -1358,6 +1358,7 @@ function selpopup(selobj) {
       </li>
       <li><a class="collapsed_lv2"><span><?php xl('Visits','e') ?></span></a>
         <ul>
+          <?php  genMiscLink('RTop','rep','0',xl('Daily Summary Report'),'reports/daily_summary_report.php'); ?>
```

Comment:
We have used SQL statements to pass the custom SQL as a bind parameters.

Please see the below file
https://github.com/rishabhemr/daily_summary_report/commit/fbdb19ab2d2e9bfc6f1b4bce8cc53dead6af684b